src/script/chem/molfile/v3000.js

```diff
@@ -123,10 +123,10 @@ function v3000parseCollection(ctab, ctabLines, shift) {
 			ctab.enhancedStereo.abs = parseBracedNumberList(prop['ATOMS'], -1);
 		} else if (line.includes("STERAC")) {
 			const n = line.substring(line.indexOf('STERAC') + 6, line.indexOf('ATOMS')).trim();
-			ctab.enhancedStereo.rac[parseInt(n, 10)] = parseBracedNumberList(prop['ATOMS'], -1);
+			ctab.enhancedStereo.rac.set(parseInt(n, 10), parseBracedNumberList(prop['ATOMS'], -1));
```

Comment:
Since enhancedStereo is derived from [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), we should be using `set` to assign values instead of the array like notation.